vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -99,11 +99,11 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			errno = err.(syscall.Errno)
 			msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 			if retryCount >= maxRetryCount {
```

Comment:
I thought with this change, we no longer need retry logic and we don't have urgency to fix #501